src/components/Node/Node.wrapper.tsx

```diff
@@ -1,5 +1,7 @@
 import * as React from 'react'
+import * as ReactDOM from 'react-dom'
 import Draggable from 'react-draggable'
+import ResizeObserver from 'react-resize-observer'
```

Comment:
maybe there should be a helper that uses either native `ResizeObserver` (see `Canvas.wrapper.tsx`) or this lib?

Reply:
I cannot help with that.